applications/SaintsUndone.md

```diff
@@ -0,0 +1,193 @@
+
+_Technical Stack_
+
+Our project utilizes a robust technical stack to deliver a seamless user experience:
+
+- Front End: React JS for the user interface
+- Smart Contracts: Solidity for secure NFT minting
+- Unreal Engine: To further develop our immersive metaverse
+- Account Abstracted Login: Ensuring a user-friendly login experience
+
+By leveraging these technologies, we aim to provide a visually captivating and technically advanced platform.
+
+**App Front-End Development (In-Progress):**
+
+![Saints_Undone_Background_Vechain](https://github.com/Pranav9931/grant-program/assets/64951792/e343acc3-e854-459e-9066-1a444c1af229)
+
+**Technical Details**:
+
+![IMG-20230928-WA0003](https://github.com/Pranav9931/grant-program/assets/146246333/ee532dd8-0f4b-4dfe-a1b7-fe27fe0e2a01)
+
+### Ecosystem Fit
+
+Saints Undone occupies a unique position within the VeChain ecosystem by introducing fashion NFT wearables.
+We bridge the gap between traditional art, fashion, and cutting-edge blockchain technology. The project offers:
+
+- A new form of self-expression for users within the VeChain metaverse.
+- A novel use case for VeChain's technology by enabling NFT minting and trading.
+- Collaboration opportunities with artists, fashion designers, and developers, fostering a creative community.
+
+Our project is aligned with VeChain's commitment to innovation, technology, and the development of new use cases.
+
+## Team
+
+### Team Members
+
+- Pranav Kumar
+- Yash Gupta
+- Sarvottam Kumar Mishra
+- GK Raghav
+
+### Team Background
+
+Our team comprises four dynamic members who are currently in their final year of college. We are excited to transition Saints Undone from a college project into a full-time endeavor. In addition to our academic journey, our collective achievements and experiences include:
+
+1. _Hackathon Success:_ Our team has an impressive track record of winning several national and international hackathons. Notable achievements include securing the fourth position in the ICC NIUM Next-In Hackathon and winning the EY Techathon 3.0 organized by EY India.
+
+- _Global Rank 4 - ICC NIUM Next-In Hackathon_ by International Cricket Council in 04/2023.
+- _National Winner - EY Techathon 3.0_ by EY India in 01/2023.
+- _National Winner - Electhon'23_ by the Election Commission of India in 05/2023.
+- _International Winner - Pygames_ by Microsoft in 05/2023.
+- _AIR 4 - National Entrepreneurship Challenge_ by E-Cell IIT Bombay in 01/2021.
+- _International Winner - ETHForAll_ by ETHIndia in 02/2023.
+- _International Winner - Fanton Q1'23_ by Fantom in 04/2023.
+- _International Winner - ETHDubai x5irelabs_ by ETHDubai in 03/2023.
+
+3. _Prior Streetwear Business:_ Before embarking on the Saints Undone project, our team successfully ran a streetwear business in the Indian market. This experience provides valuable insights into the fashion industry, which will be beneficial in our new venture.
+
+## Development Roadmap
+
+_Total Funds:_ $30,000 USD
```

Comment:
Could you update the format of this section to align with the [example](https://github.com/vechain/grant-program/blob/master/applications/application-template.md#overview-1)?